content/22.vaccines-trad.md

```diff
@@ -328,9 +326,16 @@ In a phase I/II trial, a two-dose regimen of NVX-CoV2373 was found to induce ant
-<!-- Is this still accurate? The URL below is no longer available. Delete? It does not seem essential. -->
-Novavax has also signed agreements with the U.K., Canada, Australia, and South Korea [@url:https://web.archive.org/web/20210301184421/https://ir.novavax.com/news-releases/news-release-details/novavax-announces-expanded-collaboration-and-license-agreement] and has projected that they will supply 1.1 billion doses to COVAX who will distribute the vaccines to countries with disadvantaged access to vaccine supplies [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html].
+
+In the phase III trial, the efficacy of Novavax's Nuvaxovid was reported to be 89.7%, with a total of 10 patients developing COVID-19 in the vaccine group versus 96 in the placebo group [@doi:10.1056/NEJMoa2107659].
+No hospitalizations or deaths were reported in the vaccine group.
+An additional phase III randomized, observer-blinded, placebo-controlled trial was conducted in the U.S. and Mexico, enrolling 29,949 participants and administering at least 1 vaccine in a 2:1 ratio from late December 2020 to late February 2021 with the same primary endpoints as the U.K. trial [@doi:10.1101/2021.10.05.21264567].
+A vaccine efficacy of 90.4% was reported based on 77 cases total, 63 of which occurred in the placebo group.
+All moderate to severe cases of COVID-19 occurred in the placebo group.
+Additionally, in both trials, the vaccine was found to be well-tolerated [@doi:10.1101/2021.10.05.21264567; @doi:10.1056/NEJMoa2107659].
+The conclusions of both trials indicate that the NVX-CoV2373 vaccine is safe and effective against COVID-19, and those who received this vaccine through the trials were considered fully vaccinated prior to the vaccine receiving an EUA [@url:https://ir.novavax.com/Novavax-Statement-on-UK-and-Mexico-Phase-3-Clinical-Trial-Participants-Considered-Fully-Vaccinated-in-the-US].
+The Novavax vaccine has since been authorized for use in several places, including the United Kingdom [@url:https://www.gov.uk/government/news/novavax-covid-19-vaccine-nuvaxovid-approved-by-mhra], the E.U. [@url:https://www.ema.europa.eu/en/news/ema-recommends-nuvaxovid-authorisation-eu], and the U.S.A [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-emergency-use-novavax-covid-19-vaccine-adjuvanted].
+<!-- Need to indicate it is EUA? What have we been doing elsewhere? Deleted the line below about the June FDA vote, the actual EUA didn't come until July. -->.
```

Comment:
I think this is OK now with the language changed to "authorized" -- does that sound OK?

Reply:
Yes, the authorized language is more precise and eliminated any possible confusion.